src/views/UserSearch/UserSearch.js

```diff
@@ -179,6 +185,10 @@ class UserSearch extends React.Component {
     this.setState({ showCashDrawerReportModal: modalState });
   }
 
+  changeFinancialTransactionsReportModalState = (modalState) => {
+    this.setState({ showFinancialTransactionsReportModal: modalState });
+  }
+
```

Comment:
It's good that you followed the established pattern here, but this list of copycat methods is starting to get long. If we add any more, we might consider coalescing them.